devsetup/scripts/standalone.sh

```diff
@@ -138,6 +138,13 @@ while [[ $(ssh -o BatchMode=yes -o ConnectTimeout=5 $SSH_OPT root@$IP echo ok) !
 done
 
 # Render Jinja2 files
+if [ ${COMPUTE_DRIVER} == 'ironic' ]; then
+    PRIMARY_RESOLV_CONF_ENTRY="192.168.130.11"  # Use crc for dns to enable resolving apps-crc.testing
+    # Add a hosts entry for the network address to speed up iptables
+    ssh $SSH_OPT root@$IP "echo ${IP%.*}.0 .localdomain >> /etc/hosts"
```

Comment:
If this /etc/hosts entry is the only thing we need to speed up iptables on Standalone, then perhaps we can use this always and go back to using `192.168.122.1` (gateway) as the DNS server always? At least on Fedora, deploying CRC will amend the systemd-resolved config so that it first proxies to the CRC host. (That's why iptables became slow in the standalone deployment.)

I think we'll need to resolve the apps-crc.testing names generally to connect the compute to the control plane anyway...

Reply:
The whole "HOST_PRIMARY_RESOLV_CONF_ENTRY" dance was added to speed up iptables, it should not be needed for anything else.

Reply:
Yes this "hacky" entry in hosts solves the same problem with iptables reverse lookups slowness.

+1 to change it in a follow up, drop the `HOST_PRIMARY_RESOLV_CONF_ENTRY` and just add a similar entry in /etc/hosts always.